dali/python/nvidia/dali/plugin/pytorch.py

```diff
@@ -204,7 +204,7 @@ def __next__(self):
 
                 pyt_tensors = dict()
                 for category in self._output_categories:
-                    pyt_tensors[category] = torch.zeros(category_shapes[category],
+                    pyt_tensors[category] = torch.empty(category_shapes[category],
```

Comment:
that is unitialized data, right?

Reply:
That is correct. As we create tensor that matches DALI output shape it should be filled by DALI data anyway, so there is no point in zeroing it after creation. Still let us see how tests works on that.